qcodes/validators/validators.py

```diff
@@ -12,12 +12,11 @@
 from typing import Dict as TDict
 from typing import Generic, Hashable
 from typing import List as TList
-from typing import Optional
+from typing import Literal, Optional
 from typing import Sequence as TSequence
```

Comment:
interesting that black/isort dont group all these imports into `from typing import (all the stuff here ....)`

Reply:
I think it's due to the import as ... use

We should probably rework those imports to just use typing.List rather than from typing import typing.List as TList but that is not for here